gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
```

Comment:
I wouldn't recommend Gradle wrapper 7.6
https://docs.gradle.org/7.6/release-notes.html#known-issues
[[#23067](https://github.com/gradle/gradle/issues/23067)] - Incremental compilation of java modules is broken with Gradle 7.6